src/OrbitGl/App.cpp

```diff
@@ -404,15 +404,15 @@ Future<void> OrbitApp::OnCaptureComplete() {
             GetCaptureData().incomplete_data_intervals().size());
 
   return main_thread_executor_->Schedule(
-      [this, sampling_profiler = std::move(post_processed_sampling_data)]() mutable {
+      [this, post_processed_sampling_data = std::move(post_processed_sampling_data)]() mutable {
         ORBIT_SCOPE("OnCaptureComplete");
         TrySaveUserDefinedCaptureInfo();
         RefreshFrameTracks();
-        GetMutableCaptureData().set_post_processed_sampling_data(sampling_profiler);
+        GetMutableCaptureData().set_post_processed_sampling_data(post_processed_sampling_data);
```

Comment:
This is missing a `std::move` here. (Or alternative the lambda doesn't need to be mutable)

Reply:
Thanks!